external/js/cothority/src/byzcoin/byzcoin-rpc.ts

```diff
@@ -75,7 +75,7 @@ export default class ByzCoinRPC implements ICounterUpdater {
         rpc.genesis = await skipchain.getSkipBlock(skipchainID);
         rpc.latest = rpc.genesis;
 
-        const ccProof = await rpc.getProof(CONFIG_INSTANCE_ID, waitMatch, interval);
+        const ccProof = await rpc.getProofFromLatest(CONFIG_INSTANCE_ID, waitMatch, interval);
```

Comment:
I didn't change this one because the RPC is created a few lines above and the latest block is populated with the genesis. That means this call of `getProofFromLatest` won't get any benefit as it will be the first time we check the full chain (and it is required as you only know the genesis).